types/redux-first-router/package.json

```diff
@@ -1,6 +1,6 @@
 {
-    "private": true,
-    "dependencies": {
-        "redux": "^3.7.0"
-    }
-}
\ No newline at end of file
+  "private": true,
+  "dependencies": {
+    "redux": "^3.8.0"
```

Comment:
as far as I can tell, redux 3.7.2 is the latest release, and this is causing the CI failure.